.github/workflows/publish_pypi.yml

```diff
@@ -212,6 +213,35 @@ jobs:
           path: ./dist/*.tar.gz
           if-no-files-found: error
 
+  publish_anaconda:
+    # This job is only of value to PyBaMM and would always be skipped in forks
+    # It is meant to run on a schedule, when manually triggered, and does not
+    # run on releases.
+    if: >-
+      (github.repository == 'pybamm-team/PyBaMM') &&
+      (github.event_name == 'workflow_dispatch' && github.event.inputs.target == 'anaconda') ||
+      (github.event_name == 'schedule')
+    name: Upload nightly wheels to Anaconda.org PyPI index
+    needs: [build_macos_and_linux_wheels, build_macos_arm64_wheels, build_windows_wheels, build_sdist]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download all artifacts
+        uses: actions/download-artifact@v4
+
+      - name: Move all package files to files/
+        run: |
+          mkdir files
+          mv windows_wheels/* linux_wheels/* macos_amd64_wheels/* macos_arm64_wheels/* files/
```

Comment:
Note: the source distribution is not uploaded to Anaconda because it does not make sense to do so. The nightly uploads are supposed to be binary-only. It is therefore not copied into the `files/` staging area for upload.